pkgs/by-name/si/sidekick/package.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenv,
+  installShellFiles,
+  buildGoModule,
+  fetchFromGitHub,
+  versionCheckHook,
+}:
+
+buildGoModule rec {
+  pname = "sidekick";
+  version = "0.6.6";
+
+  src = fetchFromGitHub {
+    owner = "MightyMoud";
+    repo = "sidekick";
+    rev = "v${version}";
+    hash = "sha256-Y/dQRix/cxV3NGqTGtRP6Bcprj0jzzOpudgm9a1UMLc=";
+  };
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  doCheck = true;
+
+  vendorHash = "sha256-35MThhVqCcIFH2oQNw6n73JqNVr2T6mXaIJMK9LiXq8=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/mightymoud/sidekick/cmd.version=${version}"
+  ];
+
+  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
+    installShellCompletion --cmd sidekick \
+      --bash <($out/bin/sidekick completion bash) \
+      --fish <($out/bin/sidekick completion fish) \
+      --zsh <($out/bin/sidekick completion zsh)
+  '';
+
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+
+  meta = {
+    description = "Bare metal to production ready in mins; your own fly server on your VPS.";
```

Comment:
https://nixos.org/manual/nixpkgs/stable/#var-meta-description

Reply:
on it

Reply:
updated @Kek5chen, could you please check again